recipe/meta.yaml

```diff
@@ -1,16 +1,17 @@
 {% set name = "plumed" %}
 {% set version = "2.5.1" %}
+{% set sha256 = "de309980dcfd6f6e0e70e138856f4bd9eb4d8a513906a5e6389f18a5af7f2eba" %}
```

Comment:
Keep this like it is below, we have a bot that auto-updates the `sha256` on new releases, so no need for your to do this in future.